<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowseAttachmentButtonBox</class>
 <widget class="QWidget" name="BrowseAttachmentButtonBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>144</width>
    <height>33</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="buttonBrowseAudioAttachments">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="icon">
      <iconset resource="../ressources.qrc">
       <normaloff>:/icons/icons/1411735338_519586-083_Music-128.png</normaloff>:/icons/icons/1411735338_519586-083_Music-128.png</iconset>
     </property>
     <property name="shortcut">
      <string>Ctrl+1</string>
     </property>
     <property name="popupMode">
      <enum>QToolButton::DelayedPopup</enum>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="buttonBrowseChordAttachments">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="icon">
      <iconset resource="../ressources.qrc">
       <normaloff>:/icons/icons/write13.png</normaloff>:/icons/icons/write13.png</iconset>
     </property>
     <property name="shortcut">
      <string>Ctrl+2</string>
     </property>
     <property name="popupMode">
      <enum>QToolButton::DelayedPopup</enum>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="buttonBrowsePDFAttachments">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string notr="true">...</string>
     </property>
     <property name="icon">
      <iconset resource="../ressources.qrc">
       <normaloff>:/icons/icons/pdf24.png</normaloff>:/icons/icons/pdf24.png</iconset>
     </property>
     <property name="shortcut">
      <string>Ctrl+3</string>
     </property>
     <property name="popupMode">
      <enum>QToolButton::DelayedPopup</enum>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
